recipe/meta.yaml

```diff
@@ -291,41 +286,74 @@ outputs:
         # as of pytorch=2.0.0, there is a bug when trying to run tests without the tools
         - tools
       commands:
-        - OMP_NUM_THREADS=4 python ./test/run_test.py || true  # [not win and not (aarch64 and cuda_compiler_version != "None")]
-        - python ./test/run_test.py  # [win]
         # Run pip check so as to ensure that all pytorch packages are installed
         # https://github.com/conda-forge/pytorch-cpu-feedstock/issues/24
         - pip check
         - python -c "import torch; print(torch.__version__)"                       # [not (aarch64 and cuda_compiler_version != "None")]
         - python -c "import torch; assert torch.backends.mkldnn.m.is_available()"  # [x86 and cuda_compiler_version == "None"]
-        # Running tests on aarch64 + cuda is not supported until we move to Alma Linux 8
-        # https://github.com/conda-forge/pytorch-cpu-feedstock/pull/267#issuecomment-2387225526
         - python -c "import torch; torch.tensor(1).to('cpu').numpy(); print('numpy support enabled!!!')"
         # At conda-forge, we target versions of OSX that are too old for MPS support
         # But if users install a newer version of OSX, they will have MPS support
         # https://github.com/conda-forge/pytorch-cpu-feedstock/pull/123#issuecomment-1186355073
         # - python -c "import torch; assert torch.backends.mps.is_available()" # [osx]
+
+        # python-version-specific library (default location in SP_DIR symlinks back to this)
         - test -f $PREFIX/lib/libtorch_python${SHLIB_EXT}     # [unix]
 
+        # a reasonably safe subset of tests that should run under 15 minutes
+        # disable hypothesis because it randomly yields health check errors
```

Comment:
Have you considered their [smoke test](https://github.com/pytorch/builder/blob/main/test/smoke_test/smoke_test.py)?

Reply:
AFAIU, the testing here is more comprehensive than the smoke test. But that shouldn't stop us from adding the smoke test in the next PR if it's considered useful (and potentially covers some aspects not covered by the choice of modules here)

Reply:
Side note, if we're bringing across changes from Anaconda's recipe to here, would you prefer to have each change in a separate PR or all in one PR?

Reply:
@danpetry: Depends a bit on the quantity of changes, and how well they're separated in terms of commits. But you can start with everything you have in mind, and we can carve out smaller PRs if necessary.